src/TensorAlgebra.jl

```diff
@@ -2,6 +2,8 @@ module TensorAlgebra
 
 export contract, contract!, eigen, eigvals, lq, left_null, qr, right_null, svd, svdvals
 
+include("MatrixAlgebra.jl")
+using .MatrixAlgebra: MatrixAlgebra
```

Comment:
```suggestion
```
If we don't plan to import any of the names explicitly, I think this isn't even necessary, defining a module makes it available:
```julia
julia> module Test
       f(x) = x
       end
Main.Test

julia> Test.f(1)
1
```